docs/install-node.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# In the ReadTheDocs build, this file is ran by Sphinx' conf.py to install
+# support for Node.js.
+
+set -e
+
+
+if [ "$READTHEDOCS" = 'True' ]; then
```

Comment:
Perhaps this check is redundant. You are already checking for the env variable before executing this file:

```python
if os.environ.get('READTHEDOCS') == 'True':
    installation_output = subprocess.getoutput('bash ' + os.path.join(docs_dir, 'install-node.sh'))
    ...
```